src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./CommentSection.css";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+import { far } from "@fortawesome/free-regular-svg-icons";
+import { fab } from "@fortawesome/free-brands-svg-icons";
+
+library.add(fas, far, fab);
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: [...this.props.comments],
+      inputText: "",
```

Comment:
Knowing that the data inside the comment section of dummy-data.js, you can inject objects with hardcoded username and add comments.